src/containers/importExamples/import_example_simulated_recording.py

```diff
@@ -0,0 +1,27 @@
+import spikeextractors as se
```

Comment:
This is example .py snippet that gets displayed in the gui instructions for importing